airbyte-integrations/connectors/destination-mariadb/destination_mariadb/common/sql/mariadb_types.py

```diff
@@ -0,0 +1,99 @@
+import numpy as np
+from sqlalchemy.dialects.postgresql.base import ischema_names
```

Comment:
Warum von den postgresql dialects die ischema names anpassen wenn du dann eine verbindung mit den mysql engine/dialects machst? 
Wenn dann
```suggestion
from sqlalchemy.dialects.mysql.base import ischema_names
```